tailwind.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  content: [
+    './src/**/*.{js,jsx,ts,tsx}',
+    './node_modules/tw-elements/dist/js/**/*.js',
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [require('tw-elements/dist/plugin')],
```

Comment:
I could not find a way to go around the global-require eslint error and your code still gives out that error. Can you add an inline exception just for this line for the error so that it works?